### PR TITLE
Sync metric names in lab 9

### PR DESCRIPTION
**Pull request by EdwinWenink**
The hyperparameter sweep is defined on the "training_accuracy_score" but in the training script no metric is logged under this name. Nothing will break as the sweep seems to default to "Accuracy", but the jobs will show N/A for training_accuracy_score. The corresponding plot will not show training_accuracy_score at all, but instead use the default Accuracy.

I'd prefer logging the metric as "training_accuracy_score" because this makes it more clear you can customize the metric name you want to optimize over.

# Module: 00
## Lab/Demo: 9

Changes proposed in this pull request:

- Fix wrong metric name

### CODE REVIEW

**Comment by madiepev** [comment]
Thank you for reporting this issue!

**Comment by YahyaMohamedYahya** [comment]
Thank you

**Comment by tilemanrenovations** [comment]
> Thank you for reporting this issue!
